docs/orchestrating-elastic-stack-applications/beat.asciidoc

```diff
@@ -362,6 +366,89 @@ rules:
 
 To deploy Beats in clusters with the Pod Security Policy admission controller enabled, or in OpenShift clusters, you must grant additional permissions to the Service Account used by the Beat Pods. Those Service Accounts must be bound to a Role or ClusterRole that has `use` permission for the required Pod Security Policy or Security Context Constraints. Different Beats and their features might require different settings set in their PSP/SCC.
 
+
+[id="{p}-beat-configuration-examples"]
+== Configuration Examples
+
+Below you can find manifests that address a number of common use cases and can be your starting point in exploring Beats deployed with ECK. These manifests are self-contained and work out-of-the-box on any non-secured Kubernetes cluster. All of them contain three-node Elasticsearch cluster and single Kibana instance. All Beat configurations set up Kibana dashboards if they are available for a given Beat and all required RBAC resources.
```

Comment:
It's unrelated to that particular PR but I'm thinking about it now: if we consider the Elasticsearch resource of each example as not ready for production (mostly because small resource reqs and small storage size), then why do we deploy 3 instances? Wouldn't 1 instance be good enough, and more in line with how this is an non-production example?

Reply:
I think one won't work with some default Beat index templates. Two seems very weird, so three seems like the right choice to me. We can discuss separately if there is a need.